docs/development.md

```diff
@@ -1,28 +1,16 @@
 # Development
 
-Add details here to give a brief overview of how to work with the provider APIs.
-Please reference any SDKs or API docs used to help build the integration here.
-
-## Prerequisites
-
-Supply details about software or tooling (like maybe Docker or Terraform) that
-is needed for development here.
-
-Please supply references to documentation that details how to install those
-dependencies here.
-
-Tools like Node.js and NPM are already covered in the [README](../README.md) so
-don't bother documenting that here.
+This integration focuses on [Terraform Cloud](https://www.terraform.io/cloud) and is using the [Terraform Cloud API](https://www.terraform.io/cloud-docs/api-docs#terraform-cloud-api-documentation) for interacting with the Terraform Cloud resources.
```

Comment:
It doesn't look like prettier has run against this file yet.  You can manually run it with `yarn format` if the husky Git hooks aren't automatically doing it for you.

Reply:
Sorry.  Put this on the wrong PR.